srcpkgs/m4ri/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'm4ri'
+pkgname=m4ri
+version=20200125
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --with-openmp"
```

Comment:
I think it's `--enable-openmp`.

Reply:
Also, I think you don't need `--prefix` because it's added by `gnu-configure` build style (see `common/environment/configure/gnu-configure-args.sh`).